packages/atlas/src/components/_comments/CommentBody/CommentBody.tsx

```diff
@@ -0,0 +1,48 @@
+import React, { useEffect, useRef, useState } from 'react'
+
+import { SvgActionChevronB, SvgActionChevronT } from '@/components/_icons'
+
+import { ExpandButton, StyledCommentText } from './CommentBody.styles'
+
+export const CommentBody = ({ children }: { children?: string }) => {
```

Comment:
Can't we just add proper type here?
```suggestion
export const CommentBody: React.FC = ({ children }) => {
```

Reply:
I want to be able to pass **only** string here, not React.ReactNode, that's why I wrote this.